include/swift/Frontend/FrontendInputsAndOutputs.h

```diff
@@ -210,6 +221,17 @@ class FrontendInputsAndOutputs {
   void forEachInputProducingSupplementaryOutput(
       llvm::function_ref<void(const InputFile &)> fn) const;
 
+  /// Assumes there is not more than one primary input file, if any.
+  /// Otherwise, you would need to call getPrimarySpecificPathsForPrimary
+  /// to tell it which primary input you wanted the outputs for.
+  ///
+  /// Must not be constructed on-the-fly because some parts of the compiler
+  /// truck in StringRef's to its components, so it must live as long as the
```

Comment:
I meant it in the sense of the second definition 11 in www.dictionary.com/browse/truck?s=t : "to have dealings" or second definition 10: "to exchange commodities, barter". But, since it is unclear--maybe that usage has gone out of fashion?--I'll 86 it. The rewording is clearer.

Regarding the apostrophe, it was there to show the plural of an abbreviation, as mentioned in www.grammar-monster.com/lessons/apostrophes_show_plural_of_abbreviations.htm .
But I'll take it out, since I assume it is not common usage in this code base.

Reply:
Ah, I guess I've heard "traffic in" for that idea more.

And right, while the abbreviation apostrophe is controversial, I think the deciding factor here is that we're not considering complete type names to be abbreviations, even if they have abbreviations in them.

Reply:
Ah, thanks for the clarification about type names not considered as abbreviations. Yeah, I think "truck in" might be a usage from the 60s. Anyway, I reworded it.